### DISCUSSION TITLE
My email is not working on mobile correctly.

### DISCUSSION THREAD

**Discussion by 914digital** [bootstrap-email/bootstrap-email#167] [Help]
Everything works perfectly on desktop. Is there a reason this isn't working on mobile? 

Is there a place to submit an html file? I don't want to drop all those lines in a discussion.

**Comment by stuyam** [comment]
Can you either post an example here in discussion or create and save the email in https://app.bootstrapemail.com/editor so I can privately take a look that way?

**Comment by 914digital** [comment]
Hey Stuart,
I figured out the issue: My phone (iphone 11) was in dark mode. It breaks
the CSS.

Thanks,
Dan

___
Dan Martino
*Digital Producer*

e: ***@***.***
p: 917.991.2249
w: 914digital.com

**Reply by stuyam** [reply]
Oh interesting... didn't know about that issue Ill have to look into that!

**Comment by 914digital** [comment]
Yeah. I think there's a media query for it. But I'm not sure how effective
it is.

___
Dan Martino
*Digital Producer*

e: ***@***.***
p: 917.991.2249
w: 914digital.com
